test/e2e_node/runner/remote/run_remote.go

```diff
@@ -476,8 +472,8 @@ func (a byCreationTime) Len() int           { return len(a) }
 func (a byCreationTime) Less(i, j int) bool { return a[i].creationTime.After(a[j].creationTime) }
 func (a byCreationTime) Swap(i, j int)      { a[i], a[j] = a[j], a[i] }
 
-// Returns a list of image names based on regex and number of previous images requested.
-func getGCEImages(imageRegex, imageFamily string, project string, previousImages int) ([]string, error) {
```

Comment:
Are we 100% sure this behavior is not used anywhere?

Reply:
I have searched both test-infra and kubernetes repo, and also chase the code history. Looks it was not used from day 1.